web/film-club-client/src/App.js

```diff
@@ -73,14 +73,16 @@ function App() {
 
   // Get first batch of film data
   const getUpcomingMovies = async () => {
-    const response = await fetch("/films/upcoming")
+    const page = (movies.length / 20) + 1
+    console.log(`getUpcomingMovies called for page: ${page}`)
+    const response = await fetch(`/films/upcoming?page=${page}`)
```

Comment:
It'll be worth looking into building the [queries programmatically](https://stackoverflow.com/questions/35038857/setting-query-string-using-fetch-get-request) at some point, but probably not a big deal here!